src/js/index.js

```diff
@@ -1,2 +1,49 @@
 import 'bootstrap/dist/css/bootstrap';
-import $ from 'jquery';
+
+class BitconFetcher {
+  constructor(location) {
+    this.target = location;
+  }
+
+  getPriceList(url) {
+    fetch(url)
```

Comment:
Better to return the promise object

Reply:
Could you explain a bit more? 
Fetch is an asynchronous API. This means the function is asynchronous and it already returns a Promise. We are handing the process function in the `then` function already. Keen to know more if you have more info on this :)